core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -566,6 +566,8 @@ class WorkspaceService(protected val ctx: RawlsRequestContext,
       .getPolicy(SamResourceTypeNames.workspace, workspaceId, SamWorkspacePolicyNames.owner, ctx)
       .map(_.memberEmails)
 
+  // Do not limit workspace deletion to V2 workspaces so that we can clean up old V1 workspaces as needed.
+  // Possibly create a counter when this is called on a v1 workspace?
```

Comment:
Will remove the comment when we decide whether or not to do this.

Reply:
I think a counter is worthwhile for both v1 and v2 workspace deletion calls, either here or in a separate PR+ticket. We don't have great observability on the frequency of calls to this API for v1 workspaces so it's either a counter or a logging statement and grepping.